### ISSUE TITLE
Build TOS 8.0.1 impossible !

### ISSUE THREAD

**Issue by JeanCazaux** [Talend/studio-se-master#902]
Bonjour, j'essai de faire la build de la branche 8.0.1M12 ?
Est-ce bien la branche stable de TOS 8 ?

Lorsque je lance "mvn clean install -U" sur le repo "studio-se-master" j'ai l'erreur suivante :
Failed to execute goal on project org.talend.designer.maven.repo.tcompv0: Could not resolve dependencies for project org.talend.studio:org.talend.designer.maven.repo.tcompv0:eclipse-plugin:8.0.1-M12: Failed to collect dependencies at org.talend.daikon-ee:format-detection-csv:jar:3.2.0: Failed to read artifact descriptor for org.talend.daikon-ee:format-detection-csv:jar:3.2.0: The following artifacts could not be resolved: org.talend.daikon-ee:format-detection-csv:pom:3.2.0 (absent): Could not transfer artifact org.talend.daikon-ee:format-detection-csv:pom:3.2.0 from/to releases (https://artifacts-zl.talend.com/nexus/content/repositories/releases/): status code: 401, reason phrase: Unauthorized (401) -> [Help 1]

[INFO] org.talend.designer.feature 8.0.1-M12 .............. SUCCESS [  1.014 s]
[INFO] org.talend.designer.filemultischemas 8.0.1-M12 ..... SUCCESS [  1.925 s]
[INFO] org.talend.designer.fileoutputxml 8.0.1-M12 ........ SUCCESS [  1.964 s]
[INFO] org.talend.designer.gefabstractmap 8.0.1-M12 ....... SUCCESS [  1.339 s]
[INFO] org.talend.designer.hdfsbrowse 8.0.1-M12 ........... SUCCESS [  1.644 s]
[INFO] org.talend.designer.mapper 8.0.1-M12 ............... SUCCESS [  3.689 s]
[INFO] org.talend.designer.maven.repo.tck 8.0.1-M12 ....... SUCCESS [  1.776 s]
[INFO] org.talend.designer.maven.repo.tcksdk 8.0.1-M12 .... SUCCESS [  5.657 s]
[INFO] org.talend.designer.maven.repo.tcompv0 8.0.1-M12 ... FAILURE [  4.889 s]
[INFO] studio-maven-repository-tos 8.0.1-M12 .............. SKIPPED
[INFO] studio-maven-repository-unzip 8.0.1-M12 ............ SKIPPED
[INFO] studio-components-dependencies 8.0.1-M12 ........... SKIPPED
[INFO] studio-tcompv1-dependencies 8.0.1-M12 .............. SKIPPED
[INFO] studio-tacokit-dependencies 8.0.1-M12 .............. SKIPPED

**Comment by vidhya03** [comment]
same for me did you find a solution

**Comment by JeanCazaux** [comment]
Yes contact me at contact@deilink.fr, I'll provide you with guidance on how to work around this problem.
